cmd/tendermint/commands/init.go

```diff
@@ -57,9 +57,13 @@ func initFilesWithConfig(config *cfg.Config) error {
 			GenesisTime:     tmtime.Now(),
 			ConsensusParams: types.DefaultConsensusParams(),
 		}
+		key, err := pv.GetPubKey()
+		if err != nil {
+			return err
```

Comment:
`errrors.Wrap(err, "failed to get pubkey from private validator")` ?

Reply:
Thanks! Done. I do not really understand where we use `cmn.ErrorWrap()` over `errors.Wrap` and vice versa. Did we establish some best-practice/guideline here I'm not aware of? (I prefer using `github.com/pkg/errors` over the common package anyways)

Reply:
I'd propose we use `errors.Wrap` and if/when we decide to switch to cmn.Error, we'll replace errors.Wrap with cmn.ErrorWrap